<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Saida</class>
 <widget class="QDialog" name="Saida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="sTSaida">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Filial :</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Usuário :</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Cliente :</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0" rowspan="3">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QLineEdit" name="lE_Filial"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLineEdit" name="lE_Usuario"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLineEdit" name="lE_Cliente"/>
                  </item>
                 </layout>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="pB_searchFilial">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                    <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                    <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="pB_searchUser">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                    <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                    <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="pB_searchClient">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</normaloff>
                    <disabledoff>:/rec/icons/pesquisar.png</disabledoff>
                    <selectedoff>:/rec/icons/pesquisar.png</selectedoff>../Documentos/Qt/Projetos/estoque-agro/icons/pesquisar.png</iconset>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Observações
de entrega</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
            <item row="2" column="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="pB_CriarPedido">
              <property name="font">
               <font>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="text">
               <string>Criar Pedido</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
